cumulusci/core/config/ScratchOrgConfig.py

```diff
@@ -136,6 +136,27 @@ def password(self):
             password = self.scratch_info["password"]
         return password
 
+    @property
+    def email_address(self):
+        email_address = self.config.get("email_address")
+        if not email_address:
+            # Obtain a default email address from Git.
```

Comment:
How about we factor this out into a utility function in cumulusci.utils that takes the name of the desired git config key as an argument. I suspect this isn't the only place we'll want to do something like that.

Reply:
👍 Will do.